model/execution/DeliveryExecutionManagerService.php

```diff
@@ -238,4 +236,60 @@ public function setExtraTime($deliveryExecutions, $extraTime = 0, $extendedTime
         return $result;
     }
 
+    public function adjustTimers(array $deliveryExecutions, $seconds)
+    {
+        $result = ['processed' => [], 'unprocessed' => []];
+
+        /** @var TimerAdjustmentServiceInterface $timerAdjustmentService */
+        $timerAdjustmentService = $this->getServiceLocator()->get(TimerAdjustmentServiceInterface::SERVICE_ID);
+
+        /** @var DeliveryExecution $deliveryExecution */
+        foreach ($deliveryExecutions as $deliveryExecution) {
+            if (is_string($deliveryExecution)) {
+                $deliveryExecution = $this->getDeliveryExecutionById($deliveryExecution);
+            }
+
+            $success = false;
+            if ($this->isCorrectDeliveryExecutionStateForAdjustment($deliveryExecution)) {
+                if ($seconds > 0) {
+                    $success = $timerAdjustmentService->increase($deliveryExecution, $seconds);
+                } else {
+                    $success = $timerAdjustmentService->decrease($deliveryExecution, abs($seconds));
+                }
+            }
+
+            if ($success) {
+                $result['processed'][$deliveryExecution->getIdentifier()] = true;
+            } else {
+                $result['unprocessed'][$deliveryExecution->getIdentifier()] = false;
```

Comment:
why uprocessed === false is true? why unprocessed = true is not unprocessed?

Reply:
Current frontend implementation expects such values, same as in `DeliveryExecutionManagerService::setExtraTime()`. I agree it's not very clear, maybe original developer though `unprocessed` is a failure and should get a `false`.